src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -390,7 +388,7 @@ impl<'b> Resolver<'b> {
                 debug!("(building reduced graph for external crate) building module {} {:?}",
                        name, vis);
                 let parent_link = ModuleParentLink(parent, name);
-                let module = self.new_module(parent_link, Some(def), true);
+                let module = self.new_module(parent_link, Some(def), DUMMY_NODE_ID);
```

Comment:
`DUMMY_NODE_ID` means that the module is from an extern crate, so its nearest normal ancestor doesn't have a node id. `normal_ancestor_id` is never used on modules from extern crates, so the the choice of node id isn't particularly relevant.